### DISCUSSION TITLE
How to get dialoghost Result

### DISCUSSION THREAD

**Discussion by OrangeANDc** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3248] [Q&A] [answered]
hi,can dialoghost.show(view) retrun result like window.showdialog() ?

**Comment by Keboo** [comment, marked as answer]
To get back a result you need to make sure that the dialog sets a result when it is closed. There are various ways of doing this, but I find the simplest to use a CommandParameter with the relay commands. You can see an example of doing that [here](https://github.com/Keboo/MaterialDesignInXaml.Examples/tree/master/DialogHost/DialogHost.WithResult).

You can also see several other DialogHost examples [here](https://github.com/Keboo/MaterialDesignInXaml.Examples/tree/master/DialogHost)
